From 263e54289d9994850b348cffeec9d0b6b5715389 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Sat, 20 Jun 2020 17:13:28 +0200
Subject: [PATCH 1/4] profiles: add the ipq807x profiles

These will be merged into the primary profiles patch when AX support gets
merged.

Signed-off-by: John Crispin <john@phrozen.org>
---
 profiles/cypress.yml | 12 ++++++++++++
 profiles/hawkeye.yml | 12 ++++++++++++
 profiles/wf188.yml   | 10 ++++++++++
 3 files changed, 34 insertions(+)
 create mode 100644 profiles/cypress.yml
 create mode 100644 profiles/hawkeye.yml
 create mode 100644 profiles/wf188.yml

diff --git a/profiles/cypress.yml b/profiles/cypress.yml
new file mode 100644
index 0000000000..4f17991607
--- /dev/null
+++ b/profiles/cypress.yml
@@ -0,0 +1,12 @@
+---
+profile: qcom_cp01_c1
+target: ipq807x
+subtarget: ipq60xx
+description: Build image for the QualComm Cypress
+_packages:
+  - opensync-tip
+feeds:
+  - name: ipq807x
+    path: ../../feeds/ipq807x
+
+
diff --git a/profiles/hawkeye.yml b/profiles/hawkeye.yml
new file mode 100644
index 0000000000..926829aaa8
--- /dev/null
+++ b/profiles/hawkeye.yml
@@ -0,0 +1,12 @@
+---
+profile: qcom_hk01
+target: ipq807x
+subtarget: ipq807x
+description: Build image for the QualCom Hawkeye
+_packages:
+  - opensync-tip
+feeds:
+  - name: ipq807x
+    path: ../../feeds/ipq807x
+
+
diff --git a/profiles/wf188.yml b/profiles/wf188.yml
new file mode 100644
index 0000000000..4eb433efd0
--- /dev/null
+++ b/profiles/wf188.yml
@@ -0,0 +1,10 @@
+---
+profile: cig_wf188
+target: ipq807x
+subtarget: ipq60xx
+description: Build image for the Cigtech WF188
+feeds:
+  - name: ipq807x
+    path: ../../feeds/ipq807x
+
+
-- 
2.25.1

